splinepy/microstructure/tiles/snappy.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 
 from splinepy.bezier import Bezier
-from splinepy.microstructure.tiles.tilebase import TileBase
+from splinepy.microstructure.tiles.tile_base import TileBase
 
 
-class SnappyTile(TileBase):
+class Snappy(TileBase):
```

Comment:
Hard to guess dimension of this one

Reply:
I added properties `dim` and `para_dim` to avoid confusion